airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/AesCbcEnvelopeEncryptionBlobDecoratorTest.java

```diff
@@ -0,0 +1,73 @@
+package io.airbyte.integrations.destination.s3;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+import java.util.Base64;
+import java.util.Base64.Decoder;
+import java.util.HashMap;
+import java.util.Map;
+import javax.crypto.spec.SecretKeySpec;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AesCbcEnvelopeEncryptionBlobDecoratorTest {
+
+  private static final Decoder BASE64_DECODER = Base64.getDecoder();
+
+  private AesCbcEnvelopeEncryptionBlobDecorator decorator;
+
+  @BeforeEach
+  public void setup() {
+    decorator = new AesCbcEnvelopeEncryptionBlobDecorator(
+        new SecretKeySpec(BASE64_DECODER.decode("oFf0LY0Zae9ksNZsPSJG8ZLGRRBUUhitaPKWRPPKTvM="), "AES"),
+        new SecretKeySpec(BASE64_DECODER.decode("9ZAVuZE8L4hJCFQS49OMNeFRGTCBUHAFOgkW3iZkOq8="), "AES"),
+        BASE64_DECODER.decode("04YDvMCXpvTb2ilggLbDJQ==")
```

Comment:
Nitpick. Currently there are multiple base64 strings across the file. It is also not obvious how some of the strings are generated in the assertions. This makes it difficult to debug, if the unit test is broken. Maybe move these encoded strings to constants, add some simple explanations, and reference them in the assertions?

For example:
```java
private static final String KEK_BASE64 = "oFf0LY0Zae9ksNZsPSJG8ZLGRRBUUhitaPKWRPPKTvM=";
private static final String CEK_BASE64 = "9ZAVuZE8L4hJCFQS49OMNeFRGTCBUHAFOgkW3iZkOq8=";
private static final String IV_BASE64 = "04YDvMCXpvTb2ilggLbDJQ==";
// encrypt CEK with KEK to get the encrypted CEK
private static final String ENCRYPTED_CEK = "Ck5u5cKqcY+bcFBrpsPHHUNw5Qx8nYDJ2Vqt6XG6kwxjVAJQKKljPv9NDsG6Ncoc";

// line 26-28
new SecretKeySpec(BASE64_DECODER.decode(KEK_BASE64), "AES"),
new SecretKeySpec(BASE64_DECODER.decode(CEK_BASE64), "AES"),
BASE64_DECODER.decode(IV_BASE64);

// line 67-68
"the_cek", ENCRYPTED_CEK,
"the_iv", IV_BASE64
```

Reply:
done